Sources/ITwinMobile/ITMApplication.swift

```diff
@@ -357,9 +396,7 @@ open class ITMApplication: NSObject, WKUIDelegate, WKNavigationDelegate {
             // The wait has to happen without blocking the main thread.
             self.backendLoadingDispatchGroup.wait()
             var url = self.getBaseUrl()
-            url += "#port=\(IModelJsHost.sharedInstance().getPort())"
-            url += "&platform=ios"
-            url += self.getUrlHashParams()
+            url += self.getUrlHashParams().toString()
```

Comment:
Can probably combine it into a single line `let` now.

Reply:
Good point. Done.